roxie/ccd/ccdfile.cpp

```diff
@@ -2617,6 +2617,7 @@ class CResolvedFile : implements IResolvedFileCreator, implements ISafeSDSSubscr
     offset_t fileSize;
     unsigned fileCheckSum;
     RoxieFileType fileType;
+    size32_t configBlockedIOSize = 0;
```

Comment:
naming: Would be clearer if this was configRandomIOSize or configRandomIOBlockSize.  I think in thor it causes confusion because it isn't clear - I will add a comment there.

Reply:
agree, will change.